index.js

```diff
@@ -12,6 +12,16 @@ var timeout = require('timed-out');
 var prependHttp = require('prepend-http');
 var lowercaseKeys = require('lowercase-keys');
 var status = require('statuses');
+var NestedError = require('nested-error-stacks');
+var util = require('util');
```

Comment:
Missed this, but it should be defined at the top along with the other core node modules.